ubireader/ubifs/output.py

```diff
@@ -167,7 +167,9 @@ def _set_file_perms(path, inode):
     verbose_log(_set_file_perms, 'perms:%s, owner: %s.%s, path: %s' % (inode['ino'].mode, inode['ino'].uid, inode['ino'].gid, path))
 
 def _set_file_timestamps(path, inode):
-    os.utime(path, (inode['ino'].atime_sec, inode['ino'].mtime_sec), follow_symlinks = False)
+    follow_symlinks = False
+    if os.name == 'nt': follow_symlinks = True
+    os.utime(path, (inode['ino'].atime_sec, inode['ino'].mtime_sec), follow_symlinks=follow_symlinks)
```

Comment:
This change seems to be unrelated. Please put it in a dedicated commit with a description explaining why you need to follow symlinks on Windows.

Reply:
Noted. Please put that in a dedicated commit.